src/chakra-components/control.tsx

```diff
@@ -227,17 +228,17 @@ export const DropdownIndicator = <
         },
         className
       )}
-      sx={sx}
+      sx={dropdownIndicatorSx}
     >
-      {children || <DownChevron sx={iconSx} />}
+      {children || <DownChevron sx={downChevronSx} />}
```

Comment:
No real changes here, just moved some types around and renamed the variables to be more consistent with the other components in the package